app/app.go

```diff
@@ -639,8 +629,9 @@ func New(
 	app.MsgGateKeeper = ante.NewMsgVersioningGateKeeper(app.configurator.GetAcceptedMessages())
 	app.MsgServiceRouter().SetCircuit(app.MsgGateKeeper)
 
-	// initialize stores
-	app.MountKVStores(keys)
+	// We only initialize the base stores that will be part of every version i.e. params
+	// (which contain the app version)
```

Comment:
> What does "(which contain the app version)" mean?

It means that the app version itself is in the params module so we have to include it because we need to know the app version to know what other stores should exist. Its a little bit of a chicken and egg problem